src/plugins/intel_npu/src/backend/include/zero_executor.hpp

```diff
@@ -61,6 +61,8 @@ class ZeroExecutor final : public IExecutor {
     }
 
 private:
+    void initilize_graph_through_command_list() const;
```

Comment:
Should be `initi**a**lize_graph_through_command_list` here?

Reply:
nice catch, yes ;d